src/EnergyPlus/DataHeatBalSurface.cc

```diff
@@ -222,6 +222,9 @@ namespace DataHeatBalSurface {
 	Array1D< Real64 > QRadSWInAbs; // Short wave radiation absorbed on inside of opaque surface
 	Array1D< Real64 > QRadLWOutSrdSurfs;  // Long wave radiation absorbed on outside of exterior surface
 
+	Array1D< Real64 > QAdditionalHeatSourceOutside; // Additional heat source term on boundary conditions at outside surface
+	Array1D< Real64 > QAdditionalHeatSourceInside; // Additional heat source term on boundary conditions at inside surface
+
```

Comment:
It would be awesome if these were just scalar member variables on the Surface data structure, instead of having to maintain yet two more arrays hanging out in global state needing to be allocated and deallocated.  I know this is a pattern that is already there, but if there are any significant changes to be made after this review, it would be great to have this become a new standard for future surface related enhancements.